kong/cache.lua

```diff
@@ -176,6 +176,7 @@ function _M.new(opts)
           end,
           broadcast = function(channel, data)
             opts.worker_events.post(channel_name, channel, data)
+            -- TODO: do we need to log a failed post?
```

Comment:
I think so — this is something that shouldn't ever happen in normal operation, right?

Reply:
worker events will also log the fact that it couldn't post, but not where the data originated from. So it still adds value I think

Reply:
added a log line